templates/elevation/abstract.html

```diff
@@ -0,0 +1,11 @@
+{% extends 'base.html' %}
+{% block content %}
+<h3>Elevation Data</h3>
+<p>These data are for use in the Northern Climate Reports tool. Currently the service endpoints query ASTER GDEM elevation data for Alaska and Western Canada. Each query fetches three values: minimum, maximum, and mean elevations derived from corresponding interpolations used to resample the source data from 30 m spatial resolution to 1 km. This technique is used to assure elevation fidelity between the original and downsampled data.</p>
```

Comment:
I think the official name of the web app is Climate Impact Reports now (despite the domain name). We probably want to change the title here.